crates/rolldown/src/utils/render_chunks.rs

```diff
@@ -30,16 +30,13 @@ pub async fn render_chunks<'a>(
       })
       .collect::<FxHashMap<ArcStr, Arc<RollupRenderedChunk>>>(),
   );
-  let result = try_join_all(assets.iter().enumerate().map(|(index, asset)| {
+  let result = try_join_all(assets.iter_mut().enumerate().map(|(index, asset)| {
     let chunks = Arc::clone(&chunks);
     async move {
-      // TODO(hyf0): To be refactor:
-      // - content should use ArcStr
-      // - plugin_driver.render_chunk should return Option<...> to be able to see if there is a return value by the plugin
       if let InstantiationKind::Ecma(ecma_meta) = &asset.kind {
```

Comment:
I remove the todo for some reason.
- content should use ArcStr. The String to ArcStr has overhead at here, here using String is better.
- return Option<...> to see if there is a return value by the plugin. It is meaningless at here. If here need to it we could to changed it at here.